src/components/Sidebar/SidebarContentMore/FeatureTypePresenter.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from '../../../utils/styles/styled';
+
+interface WrapperProps {
+  isFeatureName: string;
+}
+
+const FeatureTypeWrapper = styled.ul<WrapperProps>`
+  width: ${(props) => (props.isFeatureName ? '250px' : '370px')};
```

Comment:
`isFeatureName` 이 없는 경우도 가정한다면 인터페이스에서도 optional 속성을 주어야 하지 않나요?

```suggestion
interface WrapperProps {
  isFeatureName?: string;
}
```

저는 이런 식으로 `?`를 넣었어요~!